src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -144,7 +144,19 @@ def run(
                 return BatchResult.create(
                     self._start_time, datetime.utcnow(), [], AggregationResult({}, {}, {}), status=Status.Canceled
                 )
-            raise e
+            elif isinstance(e, PromptflowException):
+                raise e
+            else:
+                # For unexpected error, we need to wrap it to SystemErrorException.
+                # This allows us to see the stack trace inside.
+                unexpected_error = UnexpectedError(
+                    target=ErrorTarget.EXECUTOR,
```

Comment:
ErrorTarget.EXECUTOR.value

Reply:
or change this inside UnexpectedError

Reply:
![image](https://github.com/microsoft/promptflow/assets/111329184/31b97a3a-fb2c-40f8-9a1a-0013288bf5e1)
The type of traget is `ErrorTarget`.